logger/analytics/analyticslogger.go

```diff
@@ -23,13 +23,15 @@ import (
 // Logger writes to Firehose instead of the logging pipeline
 type Logger struct {
 	logger.KayveeLogger
+	errLogger       logger.KayveeLogger
 	fhStream        string
 	fhAPI           firehoseiface.FirehoseAPI
 	batch           []*firehose.Record
 	batchBytes      int
 	maxBatchRecords int
 	maxBatchBytes   int
 	mu              sync.Mutex
+	wg              sync.WaitGroup
```

Comment:
could name this `sendBatchWG` or something to indicate that it's for outstanding calls to sendbatch